doc/classes/Quaternion.xml

```diff
@@ -79,6 +79,7 @@
 		<method name="exp" qualifiers="const">
 			<return type="Quaternion" />
 			<description>
+				Returns the [url=https://en.wikipedia.org/wiki/Quaternion#Exponential,_logarithm,_and_power_functions]exponential[/url] of this quaternion.
```

Comment:
I'm not sure about linking to Wikipedia here, the link is not guaranteed to be preserved, and we don't do it for the other ones here, I'd suggest providing an explanation here instead, as the equations aren't very complicated

Reply:
Especially considering this is the code, which is trivial to explain:
```
Quaternion Quaternion::log() const {
	Quaternion src = *this;
	Vector3 src_v = src.get_axis() * src.get_angle();
	return Quaternion(src_v.x, src_v.y, src_v.z, 0);
}

Quaternion Quaternion::exp() const {
	Quaternion src = *this;
	Vector3 src_v = Vector3(src.x, src.y, src.z);
	real_t theta = src_v.length();
	src_v = src_v.normalized();
	if (theta < CMP_EPSILON || !src_v.is_normalized()) {
		return Quaternion(0, 0, 0, 1);
	}
	return Quaternion(src_v, theta);
}
```